WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1132,6 +1132,7 @@ AppViewButtonIconHover   = fill: @PrimaryBrandingColor@;
 AppViewButtonHover   = background-color: inherit;
 AppViewAddrBubbleContainer   = border:none; min-height:3rem; @BottomSeparator@
 AppViewAttBubbleContainer   = min-height:3rem; border:none;
+NoResultsTableListView   = padding: 15px; text-align:left; color: @SecondaryBrandingColor@; font-weight: 300; @FontSize-slightly-big@
```

Comment:
If 16px is okay to you, you can use `@SkinWrapperPadding@`, else leave it as it is.

Thanks